src/main/java/core/basesyntax/FileWork.java

```diff
@@ -9,6 +15,26 @@
  */
 public class FileWork {
     public String[] readFromFile(String fileName) {
-        return null;
+        if (fileName.isEmpty()) {
+            return new String[]{};
+        }
+        FileReader fr = null;
```

Comment:
```suggestion
        FileReader fileReader = null;
```
BTW, it is closable object... did you read the checklist?